apis/python/CMakeLists.txt

```diff
@@ -30,97 +30,41 @@ project(
     LANGUAGES CXX
 )
 
-# Search for libtilebdvcf in the default install directory and in the directory
-# specified by the LIBTILEDBVCF_PATH environment variable.
-set(LIBTILEDBVCF_SEARCH_PATHS 
-    "${CMAKE_CURRENT_SOURCE_DIR}/../../dist/lib"
-    "${CMAKE_CURRENT_SOURCE_DIR}/dist_links/dist/lib"
-    "$ENV{LIBTILEDBVCF_PATH}"
-)
-
-message(STATUS "Searching for libtiledbvcf in ${LIBTILEDBVCF_SEARCH_PATHS}")
-
-# TODO: Use find_package() instead of find_library() to find libtiledbvcf.
-find_library(
-    TILEDBVCF_LIB
-    NAMES tiledbvcf
-    PATHS ${LIBTILEDBVCF_SEARCH_PATHS}
-)
-
-if(NOT TILEDBVCF_LIB)
-    message(STATUS "Building libtiledbvcf from source")
-
-    file(MAKE_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/dist_links/libtiledbvcf/build")
-
-    execute_process(
-        COMMAND bash -c "\
-            ${CMAKE_COMMAND} .. && \
-            ${CMAKE_COMMAND} --build . && \
-            ${CMAKE_COMMAND} --build . --target install-libtiledbvcf"
-        WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/dist_links/libtiledbvcf/build"
-    )
+if (NOT TARGET tiledbvcf)
+    message(STATUS "DEBUG: tiledbvcf_DIR: ${tiledbvcf_DIR}")
+    # If tiledbvcf is not present it means that this script was not included as a
+    # subdirectory of the tiledbvcf library, thus we need to find it
```

Comment:
Why do we need to support that? There is already the capability in this PR to build libtiledbvcf and its python API from the same buildtree. If we do need to support that we will have to export the target with CMake config files.

Reply:
Because of Conda feedstock logic. VCF should be divided into two packages, VCF and VCF-Py where VCF-Py should not copy any of its shared libraries into the system if they are present as Conda packages, so it needs to build VCF-Py with VCF from the system.